gemrb/vcpkg_deps.cmake

```diff
@@ -11,68 +11,64 @@ MESSAGE(STATUS "")
 MESSAGE(STATUS "Configuring rules for VCPKG dependency deployment")
 
 SET(VCPKG_DATAROOT "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}")
-
-# we either want SDL1 or 2, no need to copy both
-IF(SDL_BACKEND MATCHES SDL2)
-	SET (DLL_SDL_VER 2)
-ENDIF()
-
-IF(CMAKE_BUILD_TYPE MATCHES "Debug")
-	SET(DLL_SET_DBG "d") # all the debug dll's just have 'd' somewhere in the filename if they are different at all
-	SET(DLL_DIR "${VCPKG_DATAROOT}/debug/bin/")
-ELSE()
-	UNSET(DLL_SET_DBG)
-	SET(DLL_DIR "${VCPKG_DATAROOT}/bin/")
-ENDIF()
+SET(DLL_DIR_DEBUG ${VCPKG_DATAROOT}/debug/bin)
+SET(DLL_DIR_RELEASE ${VCPKG_DATAROOT}/bin)
 
 # lists of dll files to be deployed to the build/install directory for Win32 builds
 # libcharset is just a relic of the current iconv build
-LIST(APPEND DLL_SET
-	${VCPKG_DATAROOT}/bin/SDL${DLL_SDL_VER}.dll # Cmake doesn't actually find the sdl debug libs
-	${DLL_DIR}SDL2_mixer.dll
-	${DLL_DIR}glew32.dll
-	${DLL_DIR}python27.dll
-	${DLL_DIR}OpenAL32.dll
-	${DLL_DIR}zlib${DLL_SET_DBG}1.dll
-	${DLL_DIR}vorbisfile.dll
-	${DLL_DIR}ogg.dll
-	${DLL_DIR}vorbis.dll
-	${DLL_DIR}libpng16${DLL_SET_DBG}.dll
-	${DLL_DIR}freetype${DLL_SET_DBG}.dll
-	${DLL_DIR}bz2${DLL_SET_DBG}.dll
-	${DLL_DIR}libcharset.dll
-	${DLL_DIR}libiconv.dll )
-
-FOREACH(ENTRY IN LISTS DLL_SET)
-
-	IF(NOT EXISTS ${ENTRY})
-		LIST(REMOVE_ITEM DLL_SET ${ENTRY})
-		CONTINUE()
+LIST(APPEND DLL_SET_DEBUG
+	brotlicommon.dll
+	brotlidec.dll
+	bz2d.dll
+	freetyped.dll
+	glew32d.dll
+	libcharset.dll
+	libiconv.dll
+	libpng16d.dll
+	ogg.dll
+	OpenAL32.dll
+	python27_d.dll
+	SDL.dll
+	SDL2d.dll
+	SDL2_mixer.dll
+	vorbis.dll
+	vorbisfile.dll
+	zlibd1.dll )
+
```

Comment:
Are you sure this mixing of debug and normal libraries produced working binaries? I think it didn't work for @chilvence, hence the code.

Reply:
Those are debug ones. Not all debug libraries have the “d” suffix. And some have a different one (like Python with its “_d”. That's why I've spit the lists.